src/Internal.AspNetCore.Sdk/sdk/Sdk.targets

```diff
@@ -9,6 +9,4 @@
   <Import Project="..\buildMultiTargeting\Internal.AspNetCore.Sdk.targets" Condition="'$(TargetFrameworks)' != '' AND '$(TargetFramework)' == ''" />
   <Import Project="..\build\Internal.AspNetCore.Sdk.targets" Condition="'$(TargetFramework)' != ''" />
 
-  <Import Project="DefaultItems.targets" />
```

Comment:
Will we update RepoTasks.csproj in AspNetCore i.e. add the lost Microsoft.Build.* references when moving to the new BuildTools?

Reply:
I don't believe any changes will be needed. This code is obsolete in the 3.0 branch. Nothing calls into this targets file because we stopped using Internal.AspNetCore.Sdk as an "SDK".